Documentation/gettingstarted/k8s-install-gke.rst

```diff
@@ -70,23 +70,23 @@ Deploy Cilium release via Helm:
 
     kubectl create namespace cilium
     helm install cilium |CHART_RELEASE| \\
-      --namespace cilium \\
+      --namespace kube-system \\
```

Comment:
Since we have removed problematic label that was causing cilium DS to be removed from the cluster, it is fine to have it in kube-system, what do you think @tgraf?

Reply:
The addon manager will remove any DaemonSet which is not whitelisted.

Reply:
We have duplicated the validate step for Azure CNI. We'll have to do the same for GKE for now.

Reply:
Ok. I'll do that.